app/routes/companyInterest/CompanyInterestEditRoute.js

```diff
@@ -27,7 +27,9 @@ const mapStateToProps = (state, props) => {
   const companyInterest = selectCompanyInterestById(state, {
     companyInterestId
   });
-  const semesters = selectCompanySemesters(state);
+  // TODO show an intersection of the currently active semesters, and
+  // the semester chosen when the interestform was created
```

Comment:
You wanna get this done in this PR or just merge it and get back to this later?

Reply:
Will fix in this PR. None of the things currently is important (Fixed the important stuff in the backend).

Reply:
WIll fix this at a later stage...